pyomo/contrib/solver/solvers/ipopt.py

```diff
@@ -431,16 +448,26 @@ def solve(self, model, **kwds):
                     results.termination_condition = TerminationCondition.error
                     results.solution_loader = SolSolutionLoader(None, None)
                 else:
-                    results.iteration_count = iters
-                    if ipopt_time_nofunc is not None:
+                    results.iteration_count = parsed_output_data['iters']
```

Comment:
We probably don't want to generate an exception if this dictionary is empty (e.g., no log file).

Reply:
Oop, good call. I had a check in here in earlier iterations, but I forgot, I removed it when I ran into the weird Windows bug because I was trying to diagnose how to fix it. Will add back in the earlier logic here!